connector/connect/client/jvm/pom.xml

```diff
@@ -116,49 +90,18 @@
           <shadedArtifactAttached>false</shadedArtifactAttached>
           <promoteTransitiveDependencies>true</promoteTransitiveDependencies>
           <artifactSet>
+            <!-- TODO make sure this is complete. -->
```

Comment:
Most of the shading is now done in connect-api.

Reply:
... and yes I still need to check iif the shading rules between SBT and maven are still consistent.